src/librustc_middle/mir/coverage/mod.rs

```diff
@@ -0,0 +1,24 @@
+//! Metadata from source code coverage analysis and instrumentation.
+
+/// Positional arguments to `libcore::count_code_region()`
+pub mod count_code_region_args {
+    pub const COUNTER_INDEX: usize = 0;
+    pub const START_BYTE_POS: usize = 1;
+    pub const END_BYTE_POS: usize = 2;
+}
+
+/// Positional arguments to `libcore::coverage_counter_add()` and
+/// `libcore::coverage_counter_subtract()`
+pub mod coverage_counter_expression_args {
```

Comment:
I almost wonder if these should be enums and cast to the appropriate integer size at use sites. This is probably fine though.

Reply:
I really wanted to use enums for this, and take advantage of implicit numbering, but I couldn't get past the awkward/ugly casting with indexing the vectors. ☹️ 

Ultimately I decided this was the lesser of two evils.